site/app/Resources/views/project/settings.html.twig

```diff
@@ -2,22 +2,13 @@
 
 {# include trumbowyg CSS/JS #}
 {% block stylesheets %}
-  {% stylesheets filter="scssphp,cssrewrite"
-      "assets/scss/trumbowyg.scss"
-  %}
-    <link rel="stylesheet" href="{{ asset_url }}" />
-  {% endstylesheets %}
+  <link  rel="stylesheet" href="{{ asset('build/trumbowyg.css') }}">
```

Comment:
nit: only one space

Any why not name this after the page as well like the other CSS files?